libs/sr/assets/src/index.ts

```diff
@@ -8,7 +8,24 @@ import type {
 import chars from './gen/chars'
 import lightCones from './gen/lightCones'
 import relics from './gen/relics'
-type characterAssetKey = 'icon' | 'basic_0' | 'skill_0'
+type characterAssetKey =
+  | 'icon'
+  | 'basic_0'
```

Comment:
there is also possibly basic_1, skill_1, ult_1, ult_2, etc, etc, but I guess we can add as needed